content/en/blog/2023/secure-apps-with-istio/index.md

```diff
@@ -0,0 +1,355 @@
+---
+title: "Secure Application Communications with Mutual TLS and Istio"
+description: Dive into requirements to secure application communications, mTLS and Istio to achieve
+end-to-end mTLS among your applications.
```

Comment:
```suggestion
description: |
  Dive into requirements to secure application communications, mTLS and Istio to achieve
  end-to-end mTLS among your applications.
```

This will fix the lint issue